src/aggregated/account_details.rs

```diff
@@ -0,0 +1,63 @@
+use actix::Addr;
+
+use near_client::{Query, ViewClientActor};
+use near_indexer::near_primitives;
+
+pub(crate) async fn get_account_balance(
+    view_client: &Addr<ViewClientActor>,
+    account_id: &near_primitives::types::AccountId,
+    block_height: &near_primitives::types::BlockHeight,
+) -> Result<near_primitives::types::Balance, String> {
+    get_account_view_for_block_height(view_client, account_id, block_height)
+        .await
+        .map(|account| account.amount)
+}
+
+pub(crate) async fn get_contract_code_hash(
+    view_client: &Addr<ViewClientActor>,
+    account_id: &near_primitives::types::AccountId,
+    block_height: &near_primitives::types::BlockHeight,
+) -> Result<near_primitives::hash::CryptoHash, String> {
+    get_account_view_for_block_height(view_client, account_id, block_height)
+        .await
+        .map(|account| account.code_hash)
+}
+
+async fn get_account_view_for_block_height(
+    view_client: &Addr<ViewClientActor>,
+    account_id: &near_primitives::types::AccountId,
+    block_height: &near_primitives::types::BlockHeight,
+) -> Result<near_primitives::views::AccountView, String> {
```

Comment:
@khorolets very good point, though we struggled with them on the library boundary, and here we are on the application level, so it is less painful, though I would still prefer some error type instead of `String`.

@telezhnaya Given that you are going on vacation soon, and I would prefer to merge this PR before that, I would not dive into error handling just yet. Still, consider reading this great article about error handling in Rust: https://nick.groenen.me/posts/rust-error-handling/. The author has a very good point of various use-cases, and in our case here we should be fine with `anyhow` crate.

Reply:
https://github.com/near/near-indexer-for-explorer/issues/143